lib/data_update_scripts/20200305201627_index_users_to_elasticsearch.rb

```diff
@@ -0,0 +1,11 @@
+module DataUpdateScripts
+  class IndexUsersToElasticsearch
+    def run
+      User.select(:id).find_each do |user|
+        Search::IndexToElasticsearchWorker.set(queue: :low_priority).perform_async(
```

Comment:
I choose to make these async because we have a lot of Users in production and I wanted this job to be able to finish successfully*. Using set this way allows us to override the default queue, high_priority, when we send the job to Sidekiq. This will ensure we don't backup an important queue with these background jobs. 

*Eventually I want to implement [rolling restarts for Sidekiq](https://github.com/mperham/sidekiq/wiki/Ent-Rolling-Restarts), or something similar, so we don't force quit the workers on every deploy but that is down the road.